### PR TITLE
Add Community Id as optional parameter to use as header

### PR DESCRIPTION
**Pull request by anton-dahlin**
Adding functionality to specify a Community id that will be added as header in order to support Communities calls.

### CODE REVIEW

**Review by derekendres — COMMENTED** [review]
looks fine.  need to update readme to include the parameter and version stuff

**Comment by anton-dahlin** [comment]
> looks fine. need to update readme to include the parameter and version stuff

Thank you, updated Readme and History

**Review by derekendres — APPROVED** [review]

**Comment by anton-dahlin** [comment]
Approving since pipeline failure is not due to these changes

File: OCSConnector/OCSConnector.pq [hunk 1]
```diff
@@ -59,12 +64,16 @@ OCSConnectorType = type function(
...
         url = url,
         source = if timeout = null
-        then Web.Contents(url)
-        else Web.Contents(url, [ Headers = [#"Request-Timeout" = Number.ToText(timeout)], Timeout = #duration(0,0,0,timeout)])
+        then if communityId = null 
+            then Web.Contents(url)
+            else Web.Contents(url, [ Headers = [#"Community-id" = communityId]])
+        else if communityId = null 
+            then Web.Contents(url, [ Headers = [#"Request-Timeout" = Number.ToText(timeout)], Timeout = #duration(0,0,0,timeout)])
+            else Web.Contents(url, [ Headers = [#"Request-Timeout" = Number.ToText(timeout), #"Community-id" = communityId], Timeout = #duration(0,0,0,timeout)])
```
Comment (kelseyduffy) [inline OCSConnector/OCSConnector.pq:76]: does this language not have the ability to append to an array? the nested if then else statements are hard to read, but without mutable arrays it's the only option.

Comment (anton-dahlin) [reply OCSConnector/OCSConnector.pq:76]: that is a great point, I will look into that, thank you!

Comment (anton-dahlin) [reply OCSConnector/OCSConnector.pq]: I ended up creating some variables to store duplicated values and use the Record.Combine function to create something dynamic-ish. What are your thoughts on this?

**Review by kelseyduffy — APPROVED** [review]

**Pull request merged**